alerta/auth/decorators.py

```diff
@@ -24,7 +24,7 @@ def decorated(f):
         def wrapped(*args, **kwargs):
 
             # API Key (Authorization: Key <key>)
-            if 'Authorization' in request.headers:
+            if 'Authorization' in request.headers and request.headers['Authorization'].find('Key ', 0, 4) == 0:
```

Comment:
why not `... and 'Key' in and request.headers['Authorization']` ?

Reply:
Do you mean L27:
```python
if 'Authorization' in request.headers and 'Key' in request.headers['Authorization']
``` 
My thinking was that .find('Key ', 0, 4) matches from beginning and 'in' might match: Authorization: Bearer ..somebase64string...Key...morebase64 ?

Reply:
what about `... and request.headers['Authorization'].startswith('Key')` ?

Reply:
startswith works and is more readable than find('..', 0, 4) == 0 (I'm python newbie and find was first substring I found). I'll update PR.

Reply:
No problem. Thanks for updating.